checkov/terraform/checks/resource/gcp/GoogleCloudPostgreSqlLogStatement.py

```diff
@@ -0,0 +1,28 @@
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.gcp.AbsGooglePostgresqlDatabaseFlags import AbsGooglePostgresqlDatabaseFlags
+
+FLAG_NAME = 'log_statement'
+FLAG_VALUES = [
+    'ddl',
+    'mod',
+    'all'
+]
+
+
+class GoogleCloudPostgreSqlLogStatement(AbsGooglePostgresqlDatabaseFlags):
+    def __init__(self):
+        name = "Ensure GCP PostgreSQL logs SQL statements"
+        check_id = "CKV_GCP_107"
```

Comment:
```suggestion
        check_id = "CKV_GCP_111"
```

CKV_GCP_107 is taken: https://github.com/bridgecrewio/checkov/blob/master/checkov/terraform/checks/resource/gcp/CloudFunctionsShouldNotBePublic.py

Reply:
oh, I see. Funny story is that I actually also work on Cloud Functions check and it started to look like episode from The Twilight Zone 😄. I've updated with master and changed check ID. Thanks @tsmithv11 !